civis/service_client.py

```diff
@@ -68,23 +64,52 @@ def _make_request(self, method, path=None, params=None, data=None,
 
 class ServiceClient():
 
-    def __init__(self, service_id, root_path=None, swagger_path="/endpoints"):
-        return_type = 'snake'
-        self._session_kwargs = {}
+    def __init__(self, service_id, root_path=None,
+                 swagger_path="/endpoints", api_key=None,
```

Comment:
I don't think we want to have an argument for the `api_key` here, accessing only via an environment variable is a more secure interface.